src/task/script.js

```diff
@@ -303,3 +303,78 @@ function handleStatusChange(event = { target: { value: 'AVAILABLE' } }) {
 }
 
 document.getElementById('visibity-hidden').classList.remove('visibity-hidden');
+
+const assigneeEl = document.getElementById('assignee');
+
+function debounce(func, delay) {
+  let searchDelay;
+  return (...args) => {
+    if (searchDelay) {
+      clearTimeout(searchDelay);
+    }
+
+    searchDelay = setTimeout(() => {
+      func(...args);
+    }, delay);
+  };
+}
+
+async function autoComplete(searchInput) {
+  try {
+    const response = await fetch(`${API_BASE_URL}/tasks`);
```

Comment:
Why are we hitting **/tasks** API?
Please read the issue properly before proceeding with any task👇 
https://github.com/Real-Dev-Squad/website-dashboard/issues/103#issue-1052347140